workers/unity/Assets/Tests/Performance/EditmodeTests/ThisShouldRun.cs

```diff
@@ -0,0 +1,73 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+using UnityEngine;
+
+namespace Improbable.Gdk.EditmodePerformanceTests
+{
+    public class ThisShouldRun : MockBase
```

Comment:
Could this be renamed to something a bit more indicative of what we're testing here, or is this meant as a general catch-all for all passing tests?

Reply:
At the moment it exists as a marker to ensure that we can get _a_ test producing results, to check if the overall flow works